docs/build-apps.md

```diff
@@ -492,6 +492,125 @@ You can validate your response data format with the associated [json schema](/js
 
 ⚠️ iAdvize can save up to 1024 characters in each field
 
+### Conversation panel app
+Conversation panel apps extends the capabilities of the Desk by allowing our clients to embed their own apps in a dedicated panel.
+
+#### App development
+The apps that can be embedded in the Desk are web apps coded in any web compliant technology - static web page,
+dynamic web pages such as php or jsp generated, single page applications written in Angular or React, etc...
+
+These apps will be embedded in an `iframe` element in the Desk, and the web server serving these apps must allow them to run 
+in an iframe on the iAdvize domains: https://www.iadvize.com or https://ha.iadvize.com.
+This is done by configuring the `X-FRAME-OPTIONS` header.
+
+The app can communicate with the desk by using a library provided by iAdvize.
+
+To use the library an app must include a javascript bundle in the html with the following code.
+```html
+<script src="https://static.iadvize.com/conversation-panel-app-lib/1.0.0/idzcpa.umd.production.min.js"></script>
+```
+
+Then in the javascript code of the app the library can be used as follows.
+```js
+idzCpa.init().then(client => {
+  client.insertTextInComposeBox('Hello world!');
+});
+```
+
+The client is obtained via the `idzCpa.init` function that returns a Promise.
+In the `then` statement of the promise the `client` can then be used to invoke functions to interact with the desk.
+
+At present the library only allows to send some text to the compose zone with the following function.
+```ts
+function insertTextInComposeBox(value: string) {}
+```
+Please note that one iframe is created per conversation in order to keep a context for an app for each conversation.
+It is recommended to keep the app very lightweight and avoid heavy processing or streaming updates.
+
+#### Configuration
+Under the Plugins section create a Conversation Panel App and then edit the following fields:
+
+#### App name
+This is the name that will show in the toolbar button that starts your app.
+The name must be provided as a json object that contains a `default` name and then names for each language the app needs
+to support.
+
+For instance
+```json
+{"default": "Orders", "EN": "Orders", "FR":  "Commandes"}
+```
+#### Icon name
+The icon name refers to a set of predefined icons provided by iAdvize that will appear in the button that starts the app.
+
+Here is the list of available options by domain - the name must be entered in upper case.
+```
```

Comment:
I still should build you a story with these

Reply:
https://front.storybook.iadvize.net/icons/master/index.html?path=/story/special-icons-for-conversation-panel-apps--page there you go!